src/MotoROS2_AllControllers.vcxproj

```diff
@@ -89,7 +89,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='YRC1000_humble|Win32'">
     <NMakeBuildCommandLine>"$(MP_VS_Install)mpBuilder.exe" -c $(Configuration) -p "$(ProjectDir)\" -n "motoros2_yrc1000_humble" -b "$(SolutionDir)bin\$(Configuration)" -o build -i "$(MP_VS_Install)YRC1000\inc"</NMakeBuildCommandLine>
     <NMakeOutput>$(MP_VS_Install)DevTools\OnlineDownload.exe</NMakeOutput>
-    <NMakePreprocessorDefinitions>$(NMakePreprocessorDefinitions);YRC1000;MOTOROS2_MEM_TRACE_ENABLE</NMakePreprocessorDefinitions>
+    <NMakePreprocessorDefinitions>$(NMakePreprocessorDefinitions);YRC1000;MOTOROS2_MEM_TRACE_ENABLE;RCUTILS_LOG_MIN_SEVERITY=RCUTILS_LOG_MIN_SEVERITY_DEBUG</NMakePreprocessorDefinitions>
```

Comment:
I'm not sure this will work as I expect / need it to. I also haven't tested it.

Reply:
The `NMakePreprocessorDefinitions` are strictly for intellisense. The actual compiler flags are in `YRC1000_xxxCompilerArguments.mps`

Reply:
Maybe these should actually be added to the `motoplus_libmicroros_config.h`.